aws/cloudformation/standalone/access_logs/access_logs.yml

```diff
@@ -132,7 +132,7 @@ Resources:
           DATABASE: !Ref AccessLogDatabase
           TABLE: !Ref AccessLogTable
           PREFIX: !Ref BucketPrefix
-      Runtime: ruby2.7
+      Runtime: ruby3.3
```

Comment:
We cannot deploy new Ruby 2.7 lambdas, so we need to update the runtime. I have not actually tested to see if the code is valid Ruby 3.3.